.github/workflows/docusaurus-prod.yml

```diff
@@ -1,10 +1,10 @@
 name: Publish Docusaurus to prod
 
 on:
-  push:
-    branches:
-      - main
-  workflow_dispatch:
+  workflow_run:
```

Comment:
Why are we changing this? 🤔

Reply:
The idea is to publish documentation only when the Publish was successful. To avoid publishing documentation of something when the artifact publish didn't happen.